tests/core/test_snapshot.py

```diff
@@ -325,13 +325,11 @@ def test_missing_intervals_end_bounded_with_lookback(make_snapshot):
     snapshot.intervals = [(to_timestamp(start), to_timestamp(end))]
 
     execution_ts = to_timestamp("2023-01-03")
-    assert snapshot.missing_intervals(start, start, execution_time=execution_ts) == [
-        (to_timestamp(start), to_timestamp(end)),
+    assert snapshot.missing_intervals(start, start, execution_time=execution_ts) == []
+    assert snapshot.missing_intervals(start, end, execution_time=execution_ts) == [
+        (to_timestamp("2023-01-01"), to_timestamp("2023-01-02")),
+        (to_timestamp("2023-01-02"), to_timestamp("2023-01-03")),
     ]
-    assert (
-        snapshot.missing_intervals(start, start, execution_time=execution_ts, end_bounded=True)
```

Comment:
Why was this test removed?

Reply:
it didn't really add anything as end_bounded is not used for lookback calculations